PocketKit/Sources/PocketKit/MyList/ItemsList/SwiftUI/AddSavedItemView.swift

```diff
@@ -0,0 +1,111 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import Textile
+import Localization
+
+struct AddSavedItemView: View {
+    @Environment(\.dismiss)
+    private var dismiss
+
+    @State private var urlString: String = ""
+    @State private var showError: Bool = false
+
+    private let model: AddSavedItemViewModel
+
+    init(model: AddSavedItemViewModel) {
+        self.model = model
+    }
+
+    var body: some View {
+        NavigationView {
```

Comment:
since `NavigationView` is deprecated, any chance we can use `NavigationStack` here?

Reply:
Yep, updated and this fixed an iPad issue where it was showing a split view.